src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -76,6 +81,16 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
             },
             {fireImmediately: true}
         );
+
+        this.unsavedGroupsReaction = reaction(
+            ()=>this.store.unsavedGroupNamesWithOrdinal,
+            ()=>{ this.unsavedGroupsWarningDismissed = false; }
```

Comment:
should be action (see note below).   it's possible react will already wrap it in action, but i'm not sure

Reply:
mobx puts this in an action, yeah